samples/sandbox/src/main/kotlin/com/bumble/appyx/sandbox/client/container/ContainerNode.kt

```diff
@@ -96,6 +101,7 @@ class ContainerNode internal constructor(
         when (navTarget) {
             is Picker -> node(buildContext) { modifier -> ExamplesList(modifier) }
             is NavModelExamples -> NavModelExamplesNode(buildContext)
+            is SharedElementFaderExample -> SharedElementWithMovableContentExampleNode(buildContext)
```

Comment:
Should we add the two examples? Shared element transitions with and without movable content

Reply:
I can add another one yeah